### ISSUE TITLE
Select Target Platform does not include "VisionOS"

### ISSUE THREAD

**Issue by minusreality** [swift-server/vscode-swift#610]
**Describe the bug**
VisionOS is not an option in the Command "Select Target Platform"

**To Reproduce**
1. Open Visual Studio Code
2. Go to View -> Command Pallette
3. Select Swift: Select Target Platform
4. Notice that VisionOS is missing

**Expected behavior**
VisionOS listed as a platform option

**Environment**
 - OS: MacOS Sonoma 14.0
 - Swift version (output of `swift --version`): swift-driver version: 1.75.2 Apple Swift version 5.8.1 (swiftlang-5.8.0.124.5 clang-1403.0.22.11.100)
Target: arm64-apple-macosx14.0
 - Visual Studio Code version: 1.82.2 (Universal)
 - vscode-swift version: 1.6.0

**Additional context**

**Comment by adam-fowler** [comment]
I'm away just now. Will look into it when I get back next week.

**Comment by adam-fowler** [comment]
Haven't really followed the VisionOS stuff. Do you know how to install the SDK for it? It doesn't seem available via Xcode 15

**Comment by stevapple** [comment]
@adam-fowler Maybe Xcode 15 beta and ARM only?

**Comment by adam-fowler** [comment]
> @adam-fowler Maybe Xcode 15 beta and ARM only?

Oh what a pain I just deleted the betas off my M1. If someone has the beta can they tell me what is returned when they call 
```
xcrun --sdk visionos --show-sdk-path
```
I'm looking for the sdk name, plus I also need the triple for the sdk. I've searched through the swift repo but it isn't mentioned there

**Comment by fwcd** [comment]
@adam-fowler It's `xros` internally:

```
xcrun --sdk xros --show-sdk-path
```

```
/Applications/Xcode-beta.app/Contents/Developer/Platforms/XROS.platform/Developer/SDKs/XROS1.0.sdk
```

**Comment by adam-fowler** [comment]
Fixed in #643
